File: AboutScene.js
var AboutScene = function () {
};

AboutScene.prototype.onDidLoadFromCCB = function () {
	if (GameDataManager.isMusicOn) cc.AudioEngine.getInstance().playMusic("main.mp3");
	var winSize = cc.Director.getInstance().getWinSize();
    var sprite = cc.Sprite.create(MultiLanguageUtil.getI18NResourceNameFrom("about.png"));
    sprite.setPosition(cc.p(winSize.width / 2, winSize.height / 2));
    this.rootNode.addChild(sprite);

    this.rootNode.setTouchEnabled(true);
    this.rootNode.onTouchesBegan = function(touches, event) {
        this.controller.onTouchesBegan(touches, event);
        return true;
    };
};

AboutScene.prototype.onTouchesBegan = function(touches, event) {
	if (GameDataManager.isSoundEffectOn) cc.AudioEngine.getInstance().playEffect("menu.mp3");
	cc.AudioEngine.getInstance().stopMusic();
	cc.Director.getInstance().popScene();
};

File: ClearFiveStages-iOS/ClearFiveStages/Scene/AboutScene.m
#import "AboutScene.h"
#import "GameDataManager.h"
#import "SimpleAudioEngine.h"
#import "MultiLanguageUtil.h"

@implementation AboutScene

-(void) didLoadFromCCB {
    if ([GameDataManager sharedManager].isMusicOn == YES) [[SimpleAudioEngine sharedEngine] playBackgroundMusic:@"main.mp3"];
    CGSize winSize = [CCDirector sharedDirector].winSize;
    CCSprite *sprite = [CCSprite spriteWithFile:[MultiLanguageUtil getI18NResourceNameFrom:@"about.png"]];
    sprite.position = ccp(winSize.width / 2, winSize.height / 2);
    [self addChild:sprite z:1];
}

-(void) ccTouchesBegan:(NSSet *)touches withEvent:(UIEvent *)event{
	if ([GameDataManager sharedManager].isSoundEffectOn == YES) [[SimpleAudioEngine sharedEngine] playEffect:@"menu.mp3"];
	[[SimpleAudioEngine sharedEngine] stopBackgroundMusic];
	[[CCDirector sharedDirector] popScene];
}
@end
